packages/ui-rating/src/RatingIcon/index.js

```diff
@@ -95,23 +104,16 @@ class RatingIcon extends Component {
   }
 
   handleTransitionEnter = () => {
-    this.applyTheme()
+    //this.applyTheme() TODO does this need to be handled with emotion?
+    // Likely not good, I see css="[Object object]" in the debugger
```

Comment:
In this case I'd rather ask @HerrTopi or @Brailor :)

Reply:
I've tested to dynamically change the number of stars and the rating and it works fine, so it was not needed